assets/src/edit-story/components/checklist/constants.js

```diff
@@ -469,6 +475,33 @@ export const PRIORITY_COPY = {
       `${MAX_VIDEO_RESOLUTION}p`
     ),
   },
+  ampValidation: {
+    title: __('Web Story validation', 'web-stories'),
+    footer: (
+      <TranslateWithMarkup
+        mapping={{
+          a: (
+            <Link
+              href={__('https://search.google.com/test/amp', 'web-stories')}
+              rel="noreferrer"
+              target="_blank"
+              aria-label={__(
+                'Learn more by visiting Google AMP Test',
+                'web-stories'
+              )}
+              onClick={onAmpErrorClick}
+              size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.X_SMALL}
+            />
+          ),
+        }}
+      >
+        {__(
+          'Your story contains errors that make it ineligible to appear in dedicated places on Google Search and Discover. For more insight on how to resolve these errors, visit <a>Google AMP Test</a>.',
+          'web-stories'
+        )}
+      </TranslateWithMarkup>
+    ),
+  },
```

Comment:
@o-fernandez @choumx Thoughts on wording here and where we should point users to?

Reply:
My suggestion (minor edits to make it shorter and add a "may" in case some errors don't prevent distro): 

"Your story contains errors that may reduce where it is seen on Google Search and Discover. To learn more, visit <a>Google AMP Test</a>"

Reply:
thanks! will update

Reply:
Per [docs](https://developers.google.com/search/docs/advanced/appearance/enable-web-stories#google-discover), there may be additional surfaces:
> Web Stories can appear on Google across Search, Google Discover, and Google Images...

How about "can affect eligibility to appear on Google"? I.e.

"Your story contains errors that can affect eligibility to appear on Google. To learn more, visit Google AMP Test"

Reply:
This look ok? 

![Screen Shot 2021-07-27 at 2 14 50 PM](https://user-images.githubusercontent.com/10720454/127228399-21453c89-d3a5-4acd-a427-f314b238c32b.png)

Reply:
Let‘s change link to „
Web Stories Test Tool“

So „… visit the 
Web Stories Test Tool“

The docs refer to it as the 
Web Stories Google Test Tool, but that‘s a bit long

Reply:
![Screen Shot 2021-07-27 at 2 49 48 PM](https://user-images.githubusercontent.com/10720454/127232151-4af01091-8b74-46fb-9a17-031e626909a2.png)

Reply:
On second thought, the Test Tool is quite technical so perhaps we shouldn't link to it. Continued here: https://github.com/google/web-stories-wp/pull/8443#discussion_r677870166

Reply:
<img width="280" alt="Screen Shot 2021-07-28 at 2 48 36 PM" src="https://user-images.githubusercontent.com/10720454/127400731-43d9c3c4-5285-4cee-b296-3de1bf5d766e.png">


@choumx thoughts?

Reply:
Looks good to me! /cc @o-fernandez